test/fuzz_group.cpp

```diff
@@ -498,13 +535,25 @@ void parse_and_apply_instructions(std::string& in, const std::string& path, util
                 TableRef t = g.get_table(table_ndx);
                 if (t->get_column_count() > 0) {
                     size_t col_ndx = get_next(s) % t->get_column_count();
-                    bool supports_search_index = _impl::TableFriend::get_column(*t, col_ndx).supports_search_index();
+                    DataType typ = t->get_column_type(col_ndx);
 
-                    if (supports_search_index) {
+                    if (typ == type_Table) {
                         if (log) {
-                            *log << "g.get_table(" << table_ndx << ")->add_search_index(" << col_ndx << ");\n";
+                            *log << "g.get_table(" << table_ndx << ")->get_subdescriptor(" << col_ndx
+                                 << ")->add_search_index(0);\n";
```

Comment:
In this case we just go for the case with one column which is what is needed for list-of-primitives.